src/components/pegout/PegOutForm.vue

```diff
@@ -160,6 +164,97 @@ export default class PegOutForm extends Vue {
 
   isReadyToSign = false;
 
+  tourSteps = [
+    {
+      target: '#wallet-connection',
+      content: `Click to select the rsk wallet where your ${this.environmentContext.getRbtcTicker()} are stored`,
+      params: {
+        highlight: true,
+        isFirst: true,
+      },
+    },
+    {
+      target: '#amount-field',
+      content: `Input the amount you want to convert into ${this.environmentContext.getBtcTicker()}`,
+      params: {
+        highlight: true,
+        isLast: false,
+      },
+    },
+    {
+      target: '#max-btn',
+      content: 'If you want to send all your available balance in the wallet',
+      params: {
+        highlight: true,
+        isLast: false,
+      },
+    },
+    {
+      target: '#derivation-addr-btn',
+      content: 'If you want to derive your destination address click here and sign the message',
+      params: {
+        highlight: true,
+        isLast: false,
+      },
+    },
+    {
+      target: '#summary-sender-address',
+      content: `This is the address in Rootstock where the ${this.environmentContext.getRbtcTicker()} will be transferred from`,
+      params: {
+        highlight: true,
+        isLast: true,
+      },
+    },
+    {
+      target: '#summary-amount',
+      content: `This is the amount you will send to convert into ${this.environmentContext.getBtcTicker()}`,
+      params: {
+        highlight: true,
+        isLast: false,
+      },
+    },
+    {
+      target: '#summary-tx-fee',
+      content: `The estimated fee required by the network in ${this.environmentContext.getBtcTicker()}. Also called <strong>Gas</strong> `,
+      params: {
+        highlight: true,
+        isLast: false,
+      },
+    },
+    {
+      target: '#summary-btc-destination',
+      content: `This is the address where the ${this.environmentContext.getBtcTicker()} will be sent`,
+      params: {
+        highlight: true,
+        isLast: false,
+      },
+    },
+    {
+      target: '#summary-estimated-fee',
+      content: `The estimated fee required by the protocol in ${this.environmentContext.getBtcTicker()}`,
+      params: {
+        highlight: true,
+        isLast: false,
+      },
+    },
+    {
+      target: '#summary-btc-estimated-amount',
+      content: 'Based on the estimated fee and the amount transferred, this is the amount that will be in your destination address',
```

Comment:
I'll change this text for: "..., this is the estimated final amount that will be transferred to the destination address."